include/config/mousebind.h

```diff
@@ -22,6 +23,9 @@ struct mousebind {
 	/* ex: BTN_LEFT, BTN_RIGHT from linux/input_event_codes.h */
 	uint32_t button;
 
+	/* scroll direction; considered iff button is UINT32_MAX */
```

Comment:
`s/iff/if/`

Not sure about the `UINT32_MAX` here, at the very least we should use a define (e.g. something along the lines of `LAB_BTN_SCROLL` or some better name).